css/og.css

```diff
@@ -129,93 +128,170 @@
 }
 
 /* =============================== */
-/* LAYER SWITCHER */
-.ogLayerSwitcherButton {
-    /* background-clip: padding-box; */
-    height: 40px;
+/*         MENU V-BAR STYLING      */
+/* =============================== */
+
+.og-menu-bar-vertical{
+    position: fixed;
+    right: 12px;
+    top: 12px;
     width: 40px;
+    height: calc(100% - 380px);
+    background-color: rgba(64, 59, 59, 0.85);
+    border-radius: 2px;
     box-shadow: 0px 1px 4px rgba(15, 15, 15, 0.17);
     z-index: 0;
+}
+
+
+/* =============================== */
+/*         MENU ΒΤΝ STYLING        */
+/*         General Settings        */
+/* =============================== */
+
+/* All buttons have this class + OFF if they are OFF */
+.menu-btn{
```

Comment:
Sorry, I had to say that adding **og-** touches every style name we use in native controls. Please, add **og-** for all new styles.